src/renderer/components/dialogs/ViewProfile.tsx

```diff
@@ -30,16 +32,33 @@ import { selectChat } from '../helpers/ChatMethods'
 const ProfileInfoName = ({
   name,
   address,
+  lastSeen,
 }: {
   name: string
   address: string
+  lastSeen: number
 }) => {
+  const tx = useTranslationFunction()
+  const year = moment(lastSeen).format('YYYY')
+  let lastSeenString = ''
+
+  // Dates from 1970 mean that contact has never been seen
+  if (parseInt(year) <= 1970) lastSeenString = tx('last_seen_unknown')
```

Comment:
```
  if (parseInt(year) <= 1970) {
    lastSeenString = tx('last_seen_unknown')
  } else {
```

Reply:
Can you please write this as a suggestion so I can add it to the batch? The button doesn't appear in this one